deployments/cloudfoundry/tile/tile.yml

```diff
@@ -94,16 +94,16 @@ forms:
   - name: otel_proxy_http
     type: string
     label: HTTP proxy URL
-    description: URL used as the proxy URL for HTTP requests unless overridden by otel_proxy_no
+    description: URL used as the proxy URL for HTTP requests unless overridden by "Proxy exclusions"
     optional: true
 
   - name: otel_proxy_https
     type: string
     label: HTTPS proxy URL
-    description: URL used as the proxy for HTTPS requests unless overridden by otel_proxy_no
+    description: URL used as the proxy for HTTPS requests unless overridden by "Proxy exclusions"
     optional: true
 
-  - name: otel_proxy_no
+  - name: otel_proxy_exclusions
```

Comment:
Variable name underscores are converted to periods when being used in the BOSH job. (That's why the BOSH job references `otel.proxy.exclusions`, but then the Tanzu Tile references `otel_proxy_exclusions`)